.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Change log
+- Add changes
+- Here
+
+## Other notes
+Additional notes if needed
+
+## Deployment reminder
+- [ ] PR requires deployment changes
+- [ ] Deployment PR done [link here]
```

Comment:
What's the Deployment PR?

Reply:
That is the PR needed to change the configuration in the `tilauspalvelu-pipelines` repository. So basically every time we add new configuration keys. For example [this](https://dev.azure.com/City-of-Helsinki/tilavarauspalvelu/_git/tilavarauspalvelu-pipelines/pullrequest/2113).